src/ui/api-models/SignatureRequestDetailView.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Do not edit this file it is auto-generated by io-utils / gen-api-models.
```

Comment:
Should we commit these?

Reply:
I'm not sure, so I'd rather leave them committed for now.

Having them in the repository eliminates the need for additional build processes, simplifies project installation, and improves the repository's "explorability."

But it's something I'd like to consider further.